hist/histv7/inc/ROOT/RHistEngine.hxx

```diff
@@ -173,6 +173,18 @@ public:
       }
    }
 
+   /// Clone this histogram engine.
+   ///
+   /// \return the clone object
+   RHistEngine<BinContentType> Clone() const
```

Comment:
Why explicitly Clone and not the copy constructor?

Reply:
More offline discussion including @jblomer with the tendency to take a "purist" approach for now: Since we want `Clone()`, we will document the choice not to implement copy constructor and assignment. If this causes major problems, we can still implement them later on but the other way around of removing them is more difficult / impossible.